internal/pkg/action/operator_uninstall.go

```diff
@@ -157,6 +158,7 @@ func (u *OperatorUninstall) getInstallPlanResources(ctx context.Context, install
 			Version: step.Resource.Version,
 			Kind:    step.Resource.Kind,
 		})
+		obj.SetName(step.Resource.Name)
```

Comment:
:+1:

But I don't think we need to unmarshal the `step.Resource.Manifest` anymore. We can just instantiate `obj` and then set GVK and Name since that's all we need to delete the object.
```go
obj := &unstructured.Unstructured{}
obj.SetGroupVersionKind(schema.GroupVersionKind{
	Group:   step.Resource.Group,
	Version: step.Resource.Version,
	Kind:    step.Resource.Kind,
})
obj.SetName(step.Resource.Name)
```